src/main/java/uk/gov/hmcts/reform/bulkscanprocessor/tasks/processor/ZipVerifiers.java

```diff
@@ -107,7 +108,7 @@ static ZipInputStream sha256WithRsaVerification(ZipStreamWithSignature zipWithSi
 
             return zipEntries;
         } catch (IOException ioe) {
-            throw new SignatureValidationException(ioe);
```

Comment:
Has `SignatureValidationException` or its construct became retired?

Reply:
still used in 5 other places in code.